src/worker.cc

```diff
@@ -1120,6 +958,24 @@ std::string jsonify_error(const valhalla_exception_t& exception, const Api& requ
     body << (request.options().has_jsonp() ? request.options().jsonp() + "(" : "") << *json_error
          << (request.options().has_jsonp() ? ")" : "");
   }
+
+  // write a few stats about the error
+  request.mutable_info()->set_error(true);
```

Comment:
notice we also mark the request as having errored out so that later on we can send a stat that says the request was ok